bin/p2-preparer/main.go

```diff
@@ -85,6 +89,41 @@ func main() {
 		logger.WithError(err).Fatalf("Could not do initial build reality at launch: %s", err)
 	}
 
+	whiteListPods := make(map[types.PodID]bool)
+	// Keep loopinng the white list of pods in pod whitelist file, and install the non existing pods
+	// exit while the white list is empty
+	if preparerConfig.PodWhitelistFile != "" {
+		for {
+			_, err := os.Stat(preparerConfig.PodWhitelistFile)
+			// if the whitelist file does not exist, end the loop and proceed
+			if os.IsNotExist(err) {
+				logger.WithError(err).Warningf("Pod whilelist file does not exist")
+				break
+			}
+
+			podWhitelist, err := util.LoadWhitelistPods(preparerConfig.PodWhitelistFile)
+			if err != nil {
+				logger.WithError(err).WithField("path", preparerConfig.PodWhitelistFile).Fatalln("Could not read pod whitelist file")
```

Comment:
Style nit: error strings should not be capitalized. https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
I've got your point, but seems they are different cases. It should be lower cases when we are forming a err, which could be inserted in a whole error message in the log. But here with `fatalln`, we are initializing the error message. So should be capitalized.